github/github.py

```diff
@@ -140,8 +140,55 @@ def query_github(api_url_template, github_repo):
 
     return response.payload
 
-get_issues = partial(query_github, "/repos/{owner}/{repo}/issues")
-get_contributors = partial(query_github, "/repos/{owner}/{repo}/contributors")
-get_forks = partial(query_github, "/repos/{owner}/{repo}/forks")
+
 get_repo_data = partial(query_github, "/repos/{owner}/{repo}")
-get_pull_requests = partial(query_github, "/repos/{owner}/{repo}/pulls")
+
+
+def iteratively_query_github(api_url_template, github_repo):
+    """
+    Like `query_github` but return a generator by repeatedly
+    iterating until no link to next page.
+    """
+
+    is_enterprise, fqdn = get_api_fqdn(github_repo)
+    base_path = "/api/v3" if is_enterprise else ""
+    path = base_path + api_url_template.format(
+        owner=github_repo.owner,
+        repo=github_repo.repo
+    )
+
+    path = path + "?per_page=100"
```

Comment:
The value 100 is the maximum `per_page` value of the gihub api. I guess we should keep it as 100.